projects/objects/television/protos/Television.proto

```diff
@@ -31,7 +31,7 @@ Robot {
       geometry IndexedFaceSet {
```

Comment:
This proto doesn't belong to the buildings folder. Is it used by one of them or is a mistake?

Reply:
I called the PR `Convert buildings` because the majority are buildings, but it is only important that we check PROTOs we plan to convert in the `feature-enu-plan`.

I think I picked television because it was alone :) Like 20+ PROTOs before and 20+ after were selected